Pano/Views/LoginRegister/LoginRegisterView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  LoginRegisterView.swift
+//  Pano
+//
+//  Created by Serdar Onur KARADAĞ on 18.07.2023.
+//
+
+import SwiftUI
+
+struct LoginRegisterView: View {
+    @AppStorage("IS_USER_LOGIN") var isUserLogin: Bool = false
```

Comment:
My suggestion would also be to place IS_USER_LOGIN text to a centralized place since it is being used in many places and it is not the best approach for using the same string everywhere is error-prone.